tfx/extensions/google_cloud_big_query/experimental/elwc_example_gen/component/executor.py

```diff
@@ -58,18 +59,19 @@ def process(
 
 def _ConvertContextAndExamplesToElwc(
     context_feature_and_examples: Tuple[bytes, List[tf.train.Example]]
-) -> input_pb2.ExampleListWithContext:
+) -> bytes:
   """Convert context feature and examples to ELWC."""
   context_feature, examples = context_feature_and_examples
   context_feature_proto = tf.train.Example()
   context_feature_proto.ParseFromString(context_feature)
-  return input_pb2.ExampleListWithContext(
+  elwc_pb2 = input_pb2.ExampleListWithContext(
       context=context_feature_proto, examples=examples)
+  return elwc_pb2.SerializeToString()
```

Comment:
can we add deterministic=True for a deterministic partition?

Reply:
Won't this also slow down the serialization process here? Worried on performance vs correctness.